massa-execution-exports/src/types.rs

```diff
@@ -361,26 +241,12 @@ pub struct ExecutionOutput {
     /// slot trace
     #[cfg(feature = "execution-trace")]
     pub slot_trace: Option<(SlotAbiCallStack, Vec<Transfer>)>,
-}
-
-#[cfg(feature = "execution-trace")]
-#[derive(Debug, Clone)]
-/// structure describing a transfer
-pub struct Transfer {
-    /// From
-    pub from: Address,
-    /// To
-    pub to: Address,
-    /// Amount
-    pub amount: Amount,
-    /// Effective received amount
-    pub effective_received_amount: Amount,
-    /// operation id
-    pub op_id: OperationId,
-    /// success or not
-    pub succeed: bool,
-    /// Fee
-    pub fee: Amount,
+    /// Deferred credits execution (empty if execution-info feature is NOT enabled)
+    pub deferred_credits_execution: Vec<(Address, Result<Amount, String>)>,
```

Comment:
Just a question to make sure I understand well, the fields will always exist even if the feature `execution-info` isn't activated but they will be empty ?

Reply:
If the feature is not defined, the field does not exist.
Example here: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=31d2d33c65d9f35de77a00435365fbe1